libp2p/protobuf/identify.ss

```diff
@@ -0,0 +1,10 @@
+#lang :std/protobuf/proto
```

Comment:
let's call the `libp2p/protobuf` package `libp2p/pb`? This is closer to go conventions and I'd like to follow them when it makes sense to do so.